content/blog/tech-preview-oct-2019.md

```diff
@@ -0,0 +1,40 @@
++++
+categories = ["Community", "Newsletter"]
+date = "2019-10-30"
+tags = ["nats", "microservices", "technical", "features", "jetstream"]
```

Comment:
imo add `security`, `service-mesh` and `monitoring`

Reply:
added